### ISSUE TITLE
[QST]The Persistent Tile Scheduler in CUTLASS?

### ISSUE THREAD

**Issue by ziyuhuang123** [NVIDIA/cutlass#1685]
Could you please explain how the persistent tile scheduler in CUTLASS works? Does it mean that a single CTA continuously processes multiple blocks, or is the work of different kernels assigned to a single CTA?

**Comment by danielhua23** [comment]
+ me

**Comment by danielhua23** [comment]
thanks @jackkosaian , Can I simply think the sm90 tile scheduler is the same behaviour and goal as group problem visitor, which is that they find the map of block and work tile id getting best L2 locality and saturate GPU?

**Comment by jackkosaian** [comment]
Yes, there are many similarities between the SM90 scheduler and the grouped GEMM problem visitor. You can think of the SM90 scheduler as being similar to the grouped GEMM problem visitor, but with only one problem (so there is no need to figure out to which problem an output tile belongs).
